plugins/services/src/js/components/modals/CreateServiceModalForm.js

```diff
@@ -318,7 +331,18 @@ class CreateServiceModalForm extends Component {
       baseConfigCopy
     );
 
-    return CreateServiceModalFormUtil.stripEmptyProperties(newConfig);
+    const config = omit(newConfig, ["labels"]);
+    const { labels } = newConfig;
+
+    if (Object.keys(labels).length === 0) {
```

Comment:
It seems like this could also be a function, given the config returning whatever version of the config is tried to be build here